Dfe.Academies.Academisation.Core/OperationResult.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Dfe.Academies.Academisation.Core
+{
+	/// <summary>
+	/// Class used to represent a result containing a conditional create of another object,
+	/// may be a CreateResult<T> if it does, or a CommandResult if it does not.
+	/// </summary>
+	public abstract class OperationResult
```

Comment:
I don't quite understand what this is used for. Can you explain?

Reply:
So this is because we don't know whether we're creating something (in which case we want a CreateResult), or not creating something (in which case we want a CommandResult). This is a parent class of both.

I've renamed it to `CommandOrCreateResult`